dashboard/static/styles/forms.styl

```diff
@@ -19,8 +19,8 @@ textarea, input {
 
 .question-mark-tip {
     width: 18px;
-    height: 18px;
-    line-height: 18px;
+    height: @width;
```

Comment:
this makes me go ... 
![giphy](https://cloud.githubusercontent.com/assets/71928/13989817/009310bc-f0e0-11e5-860f-b46cfbc68a05.gif)